File: app/src/at-vietphan/java/vn/asiantech/internship/ui/viewpager/ViewPagerAdapter.java
package vn.asiantech.internship.ui.viewpager;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentPagerAdapter;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by vietphan on 13/12/2017.
 * Class ViewPagerAdapter
 */
public class ViewPagerAdapter extends FragmentPagerAdapter {
    private List<Fragment> mFragments = new ArrayList<>();

    public ViewPagerAdapter(FragmentManager fm, List<Fragment> fragments) {
        super(fm);
    this.mFragments=fragments;
    }

    @Override
    public Fragment getItem(int position) {
        return mFragments.get(position);
    }

    @Override
    public int getCount() {
        return mFragments.size();
    }
}

File: app/src/at-hieucap/java/vn/asiantech/internship/ui/kotlinlogin/ViewPagerAdapter.kt
package vn.asiantech.internship.ui.kotlinlogin

import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentStatePagerAdapter
import vn.asiantech.internship.models.CardLogin

/**
 * Created by tiboo on 05/02/2018.
 * Create fragment state pager adapter
 */
class gitViewPagerAdapter(val fragmentManager: FragmentManager, private val listCardLogin: ArrayList<CardLogin>) : FragmentStatePagerAdapter(fragmentManager) {
    override fun getItem(position: Int): StepLoginFragment {
        return StepLoginFragment.newInstance(listCardLogin[position])
    }

    override fun getCount(): Int {
        return listCardLogin.size
    }
}
